test/templates/spec/repl_console_spec.js

```diff
@@ -0,0 +1,55 @@
+describe("REPLConsole", function() {
+  describe("#install()", function() {
+    beforeEach(function() {
+      this.elm = document.createElement("div");
+      stageElement.appendChild(this.elm);
+    });
+
+    context("install console", function() {
+      beforeEach(function() {
+        var replConsole = new REPLConsole;
+        replConsole.install(this.elm);
+      });
+      it("should have .console", function() {
+        assert.ok(hasClass(this.elm, "console"));
+      });
+      it("should have a inner", function() {
+        assert.equal(this.elm.getElementsByClassName("console-inner").length, 1);
+      });
+      it("should have a resizer", function() {
+        assert.equal(this.elm.getElementsByClassName("resizer").length, 1);
+      });
+      it("should have a close button", function() {
+        assert.equal(this.elm.getElementsByClassName("close-button").length, 1);
+      });
+    });
+  });
+
+  describe("Console Actions", function() {
+    describe("Close Action", function() {
+      beforeEach(function() {
+        this.elm = document.createElement("div");
+        this.elmId = this.elm.id = SpecHelper.randomString();
+        stageElement.appendChild(this.elm);
+      });
+
+      context("install console", function() {
```

Comment:
Do we need that context? We already test the install console, no need to keep it here just to borrow its beforeEach functions.